ws-android/Dockerfile

```diff
@@ -5,18 +5,29 @@ FROM cloud9/workspace
 # https://github.com/lukin0110/docker-android-build
 MAINTAINER Jeremy Ellis <keyfreemusic@gmail.com>
 
-ADD ./files/workspace /home/ubuntu/workspace
+# -------------------------- Transfer from Cloud9 Github to Docker -----------------------------------------
+# The Github is at https://github.com/c9/templates/tree/master/ws-android
+
+ADD ./files /home/ubuntu
```

Comment:
We can't do this as it will include check-environment script, the way it was before was fine